phpBB/phpbb/db/migration/data/v330/remove_attachment_flash.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v330;
+
+class remove_attachment_flash extends \phpbb\db\migration\migration
+{
+	// Following constants were deprecated in 3.3
+	// and moved from constants.php to compatibility_globals.php,
+	// thus define them as class constants
+	const ATTACHMENT_CATEGORY_FLASH = 5;
+
+	protected $cat_id = array(
```

Comment:
Use short array syntax.

Reply:
This is only a copy/paste of other migration